frontend/src/pages/vpn-relay-welcome.page.tsx

```diff
@@ -0,0 +1,102 @@
+import { NextPage } from "next";
+import { useLocalization } from "@fluent/react";
+import { Layout } from "../components/layout/Layout";
+import { useRuntimeData } from "../hooks/api/runtimeData";
+import logo from "../components/layout/images/relay-logo.svg";
+import logoType from "../components/layout/images/fx-private-relay-premium-logotype-dark.svg";
+import panelArt from "./images/vpn-relay-panel-art.svg";
+import vpnLogo from "./images/mozilla-vpn-logo.svg";
+import styles from "./vpn-relay-welcome.module.scss";
+import { LinkButton } from "../components/Button";
+import { useProfiles } from "../hooks/api/profile";
+import { useRouter } from "next/router";
+import { isFlagActive } from "../functions/waffle";
+import { useEffect } from "react";
+import { authenticatedFetch } from "../hooks/api/api";
+
+const VpnRelayWelcome: NextPage = () => {
+  const { l10n } = useLocalization();
+  const profile = useProfiles();
+  const router = useRouter();
+  const runtimeData = useRuntimeData();
+
+  useEffect(() => {
+    // Tell the backend that there will probably be new subscriptions.
+    // It will then ask Firefox Accounts for up-to-date subscription information,
+    // and have that ready for the next request to `/profiles`.
+    authenticatedFetch("/accounts/profile/refresh");
+  }, []);
+
+  // redirect user if they haven't purchased vpn-relay
+  if (profile && profile.data && !profile.data[0].has_vpn) {
+    router.push("/");
+  }
+
+  return runtimeData && isFlagActive(runtimeData.data, "bundle") ? (
+    <Layout>
+      <main>
+        <div className={styles["content-container"]}>
+          <div className={styles["content-head"]}>
+            <p className={styles.headline}>
+              {l10n.getString("vpn-relay-welcome-headline")}
+            </p>
+            <p className={styles.subheadline}>
+              {l10n.getString("vpn-relay-welcome-subheadline")}
+            </p>
```

Comment:
A screen reader can't see that these are in larger fonts.

```suggestion
            <b className={styles.headline}>
              {l10n.getString("vpn-relay-welcome-headline")}
            </b>
            <h2 className={styles.subheadline}>
              {l10n.getString("vpn-relay-welcome-subheadline")}
            </h2>
```